test/unit/maybe_callback.test.js

```diff
@@ -0,0 +1,196 @@
+// @ts-nocheck
+'use strict';
+
+const sinon = require('sinon');
```

Comment:
Can we rename this file?  It's not really testing `maybeCallback`, and we actually have tests for `maybeCallback` elsewhere in the repo.

Reply:
sure, went with `api.test.js`